<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIIncludeList</class>
 <widget class="QWidget" name="SIIncludeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxDetails">
     <property name="title">
      <string>Data Topic List:</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>21</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>2</y>
         <width>251</width>
         <height>38</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolAddElem">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Create and add new data type.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset resource="../../../res/lusan.qrc">
            <normaloff>:/icons/Add Data Type</normaloff>:/icons/Add Data Type</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDeleteElem">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Delete active data type.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Delete Data Type</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolInsertElem">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Remove selected field</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Insert Field</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveUp">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Move selection up.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Move Up</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveDown">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Move selection down.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset resource="../../../res/lusan.qrc">
            <normaloff>:/icons/Move Down</normaloff>:/icons/Move Down</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>521</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableIncludes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::SelectedClicked</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::TextElideMode::ElideRight</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Include Path:</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolAddElem</tabstop>
  <tabstop>toolDeleteElem</tabstop>
  <tabstop>toolInsertElem</tabstop>
  <tabstop>toolMoveUp</tabstop>
  <tabstop>toolMoveDown</tabstop>
  <tabstop>tableIncludes</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/lusan.qrc"/>
 </resources>
 <connections/>
</ui>
